ThemingDashboards/index.html

```diff
@@ -54,8 +54,15 @@
                 case "c":
                     theme = this.cloneCurrentTheme();
                     break;
-                default:
+                case "ct":
                     theme = this.createCustomTheme();
+                    break;
+                case "cs":
+                    theme = new $.ig.MountainLightTheme();
+                    this.setVisualizationSeriesColor();
```

Comment:
This actually breaks the other selections because once you add an event handler, it is never removed. I would suggest adding a new function that clears the event handler, then call it first in the onThemeSelected function.

```js
function clearVisualizationSeriesColor() {
    revealView.onVisualizationSeriesColorAssigning = null;
}
```

```js
function onThemeSelected(theme) {

    clearVisualizationSeriesColor();

    var theme;
    switch(theme) ...
}
```